### PR TITLE
Add description field for Capability/Workloads list

### PR DESCRIPTION
**Pull request by zzxwill**
Added `description` field for workloads.

To fix #821

### CODE REVIEW

**Comment by zzxwill** [comment]
Screen Shot 2020-12-29 at 10 47 10 AM

File: dashboard/src/pages/Capability/Workloads/index.tsx [hunk 1]
```diff
@@ -41,6 +41,15 @@ export default (): React.ReactNode => {
             title: {
               dataIndex: 'name',
             },
+            description: {
+              render: (text, row) => {
+                return (
+                  <Paragraph ellipsis={{ rows: 1, expandable: true, symbol: 'more' }}>
```
Comment (majian159) [inline dashboard/src/pages/Capability/Workloads/index.tsx:47]: add 'key' fix 'Each child in a list should have a unique "key" prop.'

**Review by majian159 — CHANGES_REQUESTED** [review]

File: dashboard/src/pages/Capability/Workloads/index.tsx (hunk 1 above)
Comment (zzxwill) [reply dashboard/src/pages/Capability/Workloads/index.tsx:47]: When will this issue will happen?

Comment (majian159) [reply dashboard/src/pages/Capability/Workloads/index.tsx:47]: open browser console.  
react keys doc: https://reactjs.org/docs/lists-and-keys.html#keys

Comment (zzxwill) [reply dashboard/src/pages/Capability/Workloads/index.tsx:47]: I went through the doc but I still cloud not figure out how to fix it. Can you make another PR afterwards? Thanks.

Comment (sunny0826) [reply dashboard/src/pages/Capability/Workloads/index.tsx:47]: This problem is not related to this PR and I will fix it

**Comment by sunny0826** [comment]
LGTM

**Review by sunny0826 — APPROVED** [review]

**Review by wonderflow — APPROVED** [review]

**Pull request merged**